sale_automatic_workflow/models/sale_workflow_process.py

```diff
@@ -83,7 +83,6 @@ def _default_filter(self, xmlid):
     )
     order_filter_id = fields.Many2one(
         "ir.filters",
-        string="Order Filter",
```

Comment:
Oops, this one is needed (The string will be 'Order Filter Id')

Reply:
It is managed by odoo: fields.py line 473
self.string = (
                name[:-4] if name.endswith('_ids') else
                name[:-3] if name.endswith('_id') else name
            ).replace('_', ' ').title()